_ont/ont-cig-g-97cp.md

```diff
@@ -12,17 +12,17 @@ parent: CIG
 | Vendor/Brand     | CIG              |
 | Model            | G-97CP           |
 | ODM              | ✅               |
-| Chipset          | Realtek RTL9601D |
+| Chipset          | Realtek RTL8672 |
```

Comment:
Why? Realtek RTL8672 is the SoC, the GPON Chipset is RTL9601D.

Reply:
My bad i misread. I'll rollback that part

Reply:
Rolled back in the last commit